Amazon.Ion.ObjectMapper.Test/Utils.cs

```diff
@@ -94,5 +96,34 @@ public static IIonValue StreamToIonValue(Stream stream)
         {
             return IonLoader.Default.Load(stream).GetElementAt(0);
         }
+        
+        public static IIonValue SerializeToIonWithCustomSerializer<T>(dynamic customSerializer, T item)
```

Comment:
If this is a serializer, can it instead be an `IonSerializer<dynamic>`, or what is the expected type of customSerializer?

Reply:
I tried that but it doesn't work without changing each primitive serializer to implement `IonSerializer<dynamic>`.
ie. A string serializer would need to implement `IonSerializer<dynamic>` isntead of `IonSerializer<string>`.

Reply:
I think we can use `IonSerializer<T> customSerializer, T item` to clean this and the other utility method signatures up, even if we can't clean up the user facing code.